authentication/managing_cloud_provider_credentials/cco-short-term-creds.adoc

```diff
@@ -50,6 +50,15 @@ include::modules/cco-short-term-creds-format-aws.adoc[leveloffset=+2]
 //AWS component secret permissions requirements
 include::modules/cco-short-term-creds-component-permissions-aws.adoc[leveloffset=+2]
 
+//OLM-managed Operator support for authentication with AWS STS
+include::modules/cco-short-term-creds-aws-olm.adoc[leveloffset=+2]
+
+////
+[role="_additional-resources"]
+.Additional resources
+* xr\ef:../../operators/operator_sdk/osdk-token-auth.html#osdk-aws-sts_osdk-token-auth[CCO-based workflow for OLM-managed Operators with AWS STS]
```

Comment:
Section to be added in Alex's #64808

Reply:
Peer reviewers: if Alex;s PR beats mine in, I'll remove the `\` before merging this. Otherwise, I'll do a followup later.